source/RouteEdge.cpp

```diff
@@ -0,0 +1,39 @@
+/* RouteEdge.cpp
+Copyright (c) 2014 by Michael Zahniser
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include "RouteEdge.h"
+
```

Comment:
so, according to the c++ style guide we use:

> The std namespace
> ▶ Each .cpp file should put "using namespace std;" immediately after the #includes.

Although I am not sure if it makes sense if we do not include from std

Reply:
I don't think it does make sense to use it if no members of `std` are actually used.